### PR TITLE
Feature/hig 756 get rid of conversations background permission

### PR DESCRIPTION
**Pull request by Jungyoonlim**

### CODE REVIEW

**Comment by Jungyoonlim** [comment]
@sam5pell I commented out the welcome page, so it shouldn't show. let me know!

File: components/Conversations/Panel/PanelHeader.tsx [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-  }
-
-  const handleSettings = () => {
-    setIsSettingsActive(true); 
-  }
-
+/**
+ * 
+ * @returns 
+ */
```
Comment (sam5pell) [inline components/Conversations/Panel/PanelHeader.tsx:19]: Empty comment - you can fix/remove in a follow up

Comment (Jungyoonlim) [reply components/Conversations/Panel/PanelHeader.tsx:19]: yes, thank you

**Comment by sam5pell** [comment]
- Review and separate into smaller PRs if anything is a functional change
- Leave anything that's truly just formatting changes or file reworks

Main callout is that upon saving current convo not being redirected to that saved conversation

**Comment by Jungyoonlim** [comment]
https://github.com/highlight-ing/highlight-conversations/pull/60 -- this pr is separated (the main part of this 756!)

**Pull request closed without merging**